### ISSUE TITLE
Provide easy interface to search based on index patterns

### ISSUE THREAD

**Issue by tomsommer** [elastic/elasticsearch-php#741]
Like Kibana, offer a way to handle index patterns such as `logstash-YYYY.MM.DD` etc.: https://www.elastic.co/guide/en/kibana/current/index-patterns.html

This would make it a lot easier to search logstash indexes that are split by dates, based on a given timestamp and pattern, the library would deduct which indexes it should search (Like Kibana does)

**Comment by marcing** [comment]
Perhaps passing wildcard in 'index' param would be enough?
```
		$params = [
			'index' => 'logstash-_*'
			'body' => [
				'query' => ...
			],
			'client' => ...
		];
```
Cheers

**Comment by tomsommer** [comment]
That would work, but it would search all indexes - even those outside of a given time-period.

**Comment by ezimuel** [comment]
@tomsommer you can use also [Index patterns](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/sql-index-patterns.html#sql-index-patterns) feature available from Elasticsearch 6.5 or regular expressions using query, see [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html).
